pynumaflow/__init__.py

```diff
@@ -5,6 +5,8 @@
 if os.getenv("PYTHONDEBUG"):
     os.environ["PYTHONASYNCIODEBUG"] = "1"
 
+__version__ = "0.4.0"
```

Comment:
can't we get this info from the Python egg file using some builtin module?

Reply:
or the later wheel files.

Reply:
Done